src/templates/rule.js

```diff
@@ -383,3 +388,7 @@ export const query = graphql`
     }
   }
 `;
+
+const GreyBox = ({ content }) => {
+  return <div id="greybox test">{content}</div>;
+};
```

Comment:
This should live in `components/blocks` 
I know its just a test one but can you move it to the correct place so we can use it as a good example